src/app/space/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -1,37 +1,71 @@
-    <div class="col-xs-12 col-sm-10">
-      <fabric8-edit-space-description-widget></fabric8-edit-space-description-widget>
-    </div>
-    <div class="col-xs-4 col-xs-offset-4 col-sm-2 col-sm-offset-0">
-      <button *ngIf="this.authentication.isLoggedIn()" id="analyze-overview-add-to-space-button" class="btn btn-primary btn-lg pull-right" (click)="openForgeWizard(addSpace)">Add to space</button>
+<f8-feature-toggle featureName="newSpaceDashboard" restricted="dev">
+  <div id="analyze-overview-dashboard" class="container-fluid analyze-overview-wrapper" user-level>
+    <div class="row f8-dashboard-masthead">
+      <div class="col-sm-10">
+        <fabric8-edit-space-description-widget></fabric8-edit-space-description-widget>
+      </div>
+      <div class="col-sm-2">
+        <button *ngIf="this.authentication.isLoggedIn()" id="analyze-overview-dashboard-add-to-space-button" class="btn btn-primary btn-lg f8-dashboard-masthead-create-application pull-right" (click)="openForgeWizard(addSpace)">Create an Application</button>
+      </div>
     </div>
-  </div>
-  <div class="cards-pf">
-    <div class="row row-cards-pf">
-      <div class="col-xs-12 col-md-4">
-        <fabric8-add-codebase-widget (addToSpace)="openForgeWizard(addSpace)"></fabric8-add-codebase-widget>
+    <div class="cards-pf">
+      <div class="row row-cards-pf">
+        <div class="col-xs-12 col-md-4">
+          <fabric8-create-work-item-widget></fabric8-create-work-item-widget>
+        </div>
+        <div class="col-xs-12 col-md-8">
+          <fabric8-pipelines-widget (addToSpace)="openForgeWizard(addSpace)"></fabric8-pipelines-widget>
+          <!-- <fabric8-environment-widget></fabric8-environment-widget> -->
+        </div>
       </div>
-      <div class="col-xs-12 col-md-8">
-        <fabric8-analytical-report-widget></fabric8-analytical-report-widget>
+    </div>
+    <div class="cards-pf">
+      <div class="row row-cards-pf">
+        <div class="col-xs-12 col-md-4">
+          <fabric8-add-codebase-widget (addToSpace)="openForgeWizard(addSpace)"></fabric8-add-codebase-widget>
+        </div>
+        <div class="col-xs-12 col-md-8">
+          <fabric8-analytical-report-widget></fabric8-analytical-report-widget>
+        </div>
       </div>
     </div>
   </div>
-  <div class="cards-pf">
-    <div class="row row-cards-pf padding-top-0">
-      <div class="col-xs-12 col-md-4">
-        <fabric8-create-work-item-widget></fabric8-create-work-item-widget>
+  <div id="analyze-overview" class="container-fluid analyze-overview-wrapper" default-level>
+    <div class="row margin-top-15">
+      <div class="col-xs-12 col-sm-10">
+        <fabric8-edit-space-description-widget></fabric8-edit-space-description-widget>
+      </div>
+      <div class="col-xs-4 col-xs-offset-4 col-sm-2 col-sm-offset-0">
+        <button *ngIf="this.authentication.isLoggedIn()" id="analyze-overview-add-to-space-button" class="btn btn-primary btn-lg pull-right"
```

Comment:
I think you may want `authService` instead of `authentication`

Reply:
This was already in the code - I just replicated it. I think that's something that can be looked at when the compete swap to this new dashboard is done.